va/va.h

```diff
@@ -2707,6 +2716,15 @@ typedef struct _VAEncMiscParameterBufferMultiPassFrameSize {
     unsigned long           va_reserved[VA_PADDING_LOW];
 } VAEncMiscParameterBufferMultiPassFrameSize;
 
+/** \brief Default tuning mode */
+#define VA_ENC_TUNING_MODE_DEFAULT                      0x00000000
+/** \brief Compromise latency of video encode operations to improve quality  */
+#define VA_ENC_TUNING_MODE_HIGH_QUALITY                 0x00000001
```

Comment:
duplicate with TU1?

Reply:
This would likely be equal to `quality = 1`, yes. It can be removed if you prefer.

Reply:
yes, from my perspective, I suppose it should be removed, if we need define this tuning mode, it means it should be orthogonal with quality level.

Reply:
Removed